modules/cli-installing-cli-rpm.adoc

```diff
@@ -67,11 +73,6 @@ endif::openshift-rosa[]
 # subscription-manager repos --enable="rhocp-{product-version}-for-rhel-8-x86_64-rpms"
 ----
 +
```

Comment:
It's not showing up in the render, but should also remove the `+` that was attaching the block to the step here :)
```suggestion

```